docs/faq.rst

```diff
@@ -172,22 +172,8 @@ Can KSQL connect to an Apache Kafka cluster over SSL?
 =====================================================
 
 Yes. Internally, KSQL uses standard Kafka consumers and producers.
-The procedure to securely connect KSQL to Kafka is the same as connecting any app to Kafka.
-
-For example, you can add the following entries to the KSQL server configuration file
-(ksql-server.properties). This configuration enables KSQL to connect to a Kafka
-cluster over SSL, given a trust store that will validate the SSL certificates being used
-by the Kafka Brokers.
-
-.. code:: bash
-    security.protocol=SSL
-    ssl.truststore.location=<path to trust store that trusts broker certificates>
-    ssl.truststore.password=<trust store secret>
-
-The exact settings you will need will vary depending on the security settings the Kafka brokers
-are using and how your SSL certificates are signed. For full details, please refer to the
-`Security section of the Kafka documentation
-<http://kafka.apache.org/documentation.html#security>`__.
+The procedure to securely connect KSQL to Kafka is the same as connecting any app to Kafka. For more information,
+see :ref:`ksql-security`.
```

Comment:
Can we link to the part of the security doc that covers SSL specifically? Otherwise we're just sending the user to a large doc, where they now need to search for the right info.

(Same with the other questions).

Reply:
yep, pointing to `Configuration KSQL for Secured Apache Kafka clusters` now